lldb/source/Plugins/Process/elf-core/ThreadElfCore.cpp

```diff
@@ -556,6 +566,22 @@ Status ELFLinuxSigInfo::Parse(const DataExtractor &data, const ArchSpec &arch) {
   si_signo = data.GetU32(&offset);
   si_errno = data.GetU32(&offset);
   si_code = data.GetU32(&offset);
+  // 64b ELF have a 4 byte pad.
+  if (data.GetAddressByteSize() == 8)
+    offset += 4;
+  if (IsSignalWithAddrValue(si_signo)) {
+    addr = data.GetAddress(&offset);
+    addr_lsb = data.GetU16(&offset);
+  }
 
   return error;
 }
+
+std::string ELFLinuxSigInfo::GetDescription() {
+  if (IsSignalWithAddrValue(si_signo))
+    return lldb_private::UnixSignals::CreateForHost()->GetSignalDescription(
```

Comment:
And this shouldn't be using the *Host* UnixSignals object for the same reason. Get the one from the `Process` instance.